client/my-sites/checkout/composite-checkout/components/prepurchase-notices/akismet-product-overlaps-owned-product-notice.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { Purchase } from 'calypso/lib/purchases/types';
```

Comment:
These could both be imported as a type instead
```suggestion
import type { FunctionComponent } from 'react';
import type { Purchase } from 'calypso/lib/purchases/types';
```